modules/vorbis/resource_importer_ogg_vorbis.h

```diff
@@ -28,31 +28,35 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#ifndef RESOURCEIMPORTEROGGVORBIS_H
-#define RESOURCEIMPORTEROGGVORBIS_H
+#ifndef RESOURCEIMPORTEROPUS_H
+#define RESOURCEIMPORTEROPUS_H
 
-#include "audio_stream_ogg_vorbis.h"
 #include "core/io/resource_importer.h"
 
-class ResourceImporterOGGVorbis : public ResourceImporter {
-	GDCLASS(ResourceImporterOGGVorbis, ResourceImporter);
+class ResourceImporterOggVorbis : public ResourceImporter {
+	GDCLASS(ResourceImporterOggVorbis, ResourceImporter);
+
+	enum {
+		OGG_SYNC_BUFFER_SIZE = 8192,
+	};
+
+private:
+	// virtual int get_samples_in_packet(Vector<uint8_t> p_packet) = 0;
 
 public:
-	virtual String get_importer_name() const override;
-	virtual String get_visible_name() const override;
 	virtual void get_recognized_extensions(List<String> *p_extensions) const override;
 	virtual String get_save_extension() const override;
 	virtual String get_resource_type() const override;
-
+	virtual String get_importer_name() const override;
+	virtual String get_visible_name() const override;
 	virtual int get_preset_count() const override;
 	virtual String get_preset_name(int p_idx) const override;
-
 	virtual void get_import_options(List<ImportOption> *r_options, int p_preset = 0) const override;
 	virtual bool get_option_visibility(const String &p_option, const Map<StringName, Variant> &p_options) const override;
 
 	virtual Error import(const String &p_source_file, const String &p_save_path, const Map<StringName, Variant> &p_options, List<String> *r_platform_variants, List<String> *r_gen_files = nullptr, Variant *r_metadata = nullptr) override;
 
-	ResourceImporterOGGVorbis();
+	ResourceImporterOggVorbis();
 };
 
-#endif // RESOURCEIMPORTEROGGVORBIS_H
+#endif // RESOURCEIMPORTEROPUS_H
```

Comment:
This isn't Opus.

Reply:
Yeah, sorry. This came out of some work I was doing to support opus and some stuff didn't get changed. It's still a draft, not really ready for a deep review yet.